lib/kramdown-rfc2629.rb

```diff
@@ -833,14 +833,25 @@ def get_and_cache_resource(url, cachefile, tvalid = 7200, tn = Time.now)
         end
       end
 
+      def self.bcp_std_ref(t, n)
+        warn "*** #{t} anchors not supported in v2 format" unless $options.v3
+        "https://xml2rfc.tools.ietf.org/public/rfc/bibxml-rfcsubseries/reference.##{t}.#{"%04d" % n.to_i}.xml"
```

Comment:
Any reason why this doesn't use XML_RESOURCE_ORG_PREFIX?
Also, rfcsubseries vs. subseries above?

Reply:
These appear to be stuck at December 2019.

Reply:
I forgot the prefix part in the other file.  As for the thing, there is a "bibxml-rfcsubseries-new" that seems to be updating.  That greatly diminishes my confidence in the resource being valid though...